packages/ecs-agent/README.md

```diff
@@ -0,0 +1,23 @@
+# ecs-agent
+
+The ecs-agent package in Bottlerocket provides the ECS agent and a systemd unit
+that sets up necessary configuration on the host.
+
+This README is temporary and is meant to track the known issues and remaining
+work items for the ECS agent.
+
+## Known issues
```

Comment:
nit: two other things that i can see that remain to be addressed and not mentioned here:
1. log path of cni plugin, currently all of them are defaulting to the container path of the log directory bind mount,  e.g.  [here](https://github.com/aws/amazon-ecs-cni-plugins/blob/0c6216c60401232805e50e31d4040ae84b0b23cf/plugins/eni/main.go#L32), [here](https://github.com/aws/amazon-ecs-agent/blob/master/agent/ecscni/plugin.go#L39)
2. path to process's network namespace handle currently hardcoded to the container path of the procfs bind mount https://github.com/aws/amazon-ecs-agent/blob/782948476da6d995ad33c6a04130f8172820af27/agent/ecscni/types.go#L38

would be good to track them

Reply:
Sure, I can add those.  However, none of the CNI-related features are currently supported as is called out in the second bullet point.

Reply:
Done